homeassistant/components/zwave_js/manifest.json

```
@@ -30,5 +30,8 @@
     }
   ],
   "zeroconf": ["_zwave-js-server._tcp.local."],
-  "loggers": ["zwave_js_server"]
+  "loggers": ["zwave_js_server"],
+  "supported_brands": {
+    "leviton": "Leviton Z-Wave"
```

Shouldn't the key also have the suffix?
```suggestion
    "leviton_z_wave": "Leviton Z-Wave"
```